sites/hashdev/src/components/MdxPre.tsx

```diff
@@ -1,32 +1,35 @@
 import { Box } from "@mui/material";
-import { BoxProps } from "@mui/system";
+import { ReactElement } from "react";
+
+import { Snippet } from "./Snippet";
 
 /**
  * @todo copy button
  */
-export const MdxPre = ({ children, ...props }: BoxProps<"pre">) => {
+export const MdxPre = ({ children: codeEl }: { children: ReactElement }) => {
+  const { className, children } = codeEl.props;
```

Comment:
this would crash if you had a `<pre>` without a `<code>` inside it but I'm not sure that's possible in MDX, since ``` will automatically be structured like that